### PR TITLE
Add RWTS-PDFwriter v1.0.1

### PR DESCRIPTION
**Pull request by jnooree**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

File: Casks/rwts-pdfwriter.rb [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+
+  url "https://github.com/rodyager/RWTS-PDFwriter/releases/download/v#{version}/RWTS-PDFwriter.pkg"
+  name "RWTS PDFwriter"
+  desc "Print driver for printing documents directly to a pdf file"
+  homepage "https://github.com/rodyager/RWTS-PDFwriter"
+
+  livecheck do
+    url :url
+    strategy :github_latest
+  end
```
Comment (SMillerDev) [inline Casks/rwts-pdfwriter.rb:13]: It doesn't seem like they have pre-releases so this is not needed.
```suggestion
```

Comment (jnooree) [reply Casks/rwts-pdfwriter.rb]: Thank you for comments!

**Comment by jnooree** [comment]
The `/Users/Shared/PDFwriter` directory is actually a symlink to `/var/spool/pdfwriter`. One complication here: it turns out that subdirectories under it contain icon files(`/var/spool/pdfwriter/<user>/Icon
`), so they are not empty even right after a fresh install.

It seems just deleting the files isn't a good idea. Any suggestions?

**Comment by neutric** [comment]
> It seems just deleting the files isn't a good idea. Any suggestions?

They wouldn't get deleted with `uninstall rmdir` and doing anything else would violate our rule to not touch user-created files. In this here case, `uninstall rmdir` serves as (not very straight-forward) documentation that there is another file location to think about.

**Pull request merged**